api/redaction/markup.ts

```diff
@@ -0,0 +1,70 @@
+import * as express from 'express'
+import { getConfigValue } from '../configuration'
+import { SERVICES_MARKUP_API_URL } from '../configuration/references'
+import { EnhancedRequest } from '../lib/models'
+import { Redaction } from './redactionModels'
+import { handleDelete, handleGet, handlePost, handlePut } from './redactionService'
+
+const url: string = getConfigValue(SERVICES_MARKUP_API_URL)
+
+/**
+ * getMarkup
+ */
+export async function getMarkup(req: EnhancedRequest, res: express.Response, next: express.NextFunction) {
+
+  const markupPath: string = url + req.originalUrl
+
+  try {
+    const {status, data}: {status: number, data: Redaction[]} = await handleGet(markupPath, req)
```

Comment:
Wowsers! I didn't know you could do this!

Reply:
yeah that's pretty cool, can type the spread